### DISCUSSION TITLE
QuickAdd - how to invoke?

### DISCUSSION THREAD

**Discussion by mlevison** [chrisgrieser/shimmering-obsidian#155] [Q&A]
I can see from the codebase QuickAdd is mentioned and yet I can't see it in the Alfred Interface.

**Comment by chrisgrieser** [comment]
Where in the code base do you see that? There was a quick add feature ages ago, the workflow does not have any feature concerning quickadd anymore

**Comment by mlevison** [comment]
Info.plist - tell application "Obsidian" to open location "obsidian://advanced-uri?commandid=quickadd%253ArunQuickAdd" -- line 3650

**Reply by chrisgrieser** [reply]
yes, that's just some code Alfred has left over and did not clean up
